collection-scripts/gather

```diff
@@ -117,5 +117,8 @@ oc adm inspect --dest-dir must-gather --rotated-pod-logs "${all_ns_resources[@]}
 # Gather Performance profile information
 /usr/bin/gather_ppc
 
+# Gather SNO resources
+/usr/bin/gather_sno
```

Comment:
Do we want to run this all the time (even on non SNO clusters)?

Reply:
I thought that workload partitioning is an SNO's only feature. but now I understand it being [introduced for multi-node clusters from OCP 4.13](https://cloud.redhat.com/blog/your-guide-to-workload-partitioning-for-multi-node-clusters-in-openshift-4.13). so maybe the "gather_sno" naming should be changed.
Anyhow, we want to run this script for each node, on a cluster that has workload partitioning enabled.